### PR TITLE
theam button fix #647

### LINKED ISSUE #647: Theme button position needs to be changed
### Description

Currently theme button is on screen and we are unable to change its position, and also in mobile view it anoying.

### Steps to reproduce

We can see theme button position that needs to changed 

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by biplavmz**
bug issue #647 Theam button position change and icon change.
image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -173,7 +174,13 @@ function App() {
             postText={postText}
           />
 
-          <Darkmode />
+          {location.pathname == "/dummygram/login" && (
+            <Darkmode theamClass="theamButton theamButton-login" />
+          )}
+          {location.pathname == "/dummygram/signup" && (
+            <Darkmode theamClass="theamButton theamButton-login" />
```
Comment (narayan954) [inline src/App.jsx:181]: broo, please use proper spellings, theme instead of theam, all over

Comment (narayan954) [inline src/App.jsx:181]: also, why is this duplicate, you could've just used `or` operator

File: src/reusableComponents/Darkmode/index.jsx [hunk 2]
```diff
@@ -29,19 +32,32 @@ const Darkmode = () => {
...
+
   return (
-    <div className="darkmode-btn">
-      <button onClick={darkModeToggle}>
-        <span>
-          <span></span>
-        </span>
-      </button>
+    <div className="">
+      <div className="">
```
Comment (narayan954) [inline src/reusableComponents/Darkmode/index.jsx:50]: empty classnames?

File: src/index.css [hunk 3]
```diff
@@ -65,19 +65,7 @@ body {
...
-  flex-wrap: wrap;
-  justify-content: space-between;
-  object-fit: contain;
-  padding-bottom: 10px;
-  padding-top: 10px;
-  position: fixed;
-  top: 0;
-  width: 100%;
-  z-index: 1;
+
```
Comment (narayan954) [inline src/index.css:79]: empty style?

**Review by narayan954 — CHANGES_REQUESTED** [review]
please fix the naming conventions

**Comment by biplavmz** [comment]
ok @narayan954

**Pull request closed without merging**